platform-emf/src/main/java/com/softicar/platform/emf/form/EmfFormSaveOrCancelActionsInput.java

```diff
@@ -53,23 +53,16 @@ public EmfFormSaveOrCancelActionsInput(IEmfFormBody<R> formBody, EmfFormAttribut
 	private void save(boolean closeAfterSave) {
 
 		boolean creation = formBody.getTableRow().impermanent();
-		try (DbTransaction transaction = new DbTransaction()) {
+		try (DbTransaction rootTransaction = new DbTransaction()) {
```

Comment:
Totally optional but I kind of like this to reduce redundancy:
```suggestion
		try (var rootTransaction = new DbTransaction()) {
```